File: src/dataset/javascript/rna-transcription_exercism-javascript-PhymasSC/claude/ClaudeSonnet4_rna-transcription_v4.js
const DNA_TO_RNA = {
	G: "C",
	C: "G",
	T: "A",
	A: "U"
};

export const toRna = dnaSequence => {
	if (!dnaSequence) return "";
	
	let result = "";
	for (let i = 0; i < dnaSequence.length; i++) {
		result += DNA_TO_RNA[dnaSequence[i]];
	}
	return result;
};

File: src/dataset/cpp/rna_transcription_Exercism_MaxyMoos/claude/ClaudeSonnet4_rna_transcription_v4.cpp
#include "rna_transcription.h"

using namespace std;

char transcription::to_rna(char input)
{
    return transTable[input];
}

string transcription::to_rna(string input)
{
    string result;
    result.reserve(input.size());
    for (const char& c : input)
    {
        result += transTable[c];
    }
    return result;
}